CHANGELOG.md

```diff
@@ -1,5 +1,77 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-beta.4
+
+<!-- generated comparing v5.0.0-beta.3..next -->
+
+_Aug 13, 2021_
+
+Big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- Grid's "auto" behaviour has been fixed by @aaronlademann-wf (#27514)
+- @ryancogswell contribution fixed navigation in MenuList (#27526)
```

Comment:
```suggestion
- @ryancogswell contribution fixed keyboard navigation in MenuList (#27526)
```
Technically it's just "text-based keyboard navigation" but let's be a bit more vague so that people see the urgency of this update